src/nvidia_rag/utils/summarization.py

```
@@ -0,0 +1,975 @@
+    from crashed or restarted processes. Runs synchronously on module import.
+    """
+    if not SUMMARY_STATUS_HANDLER.is_available():
+        logger.debug("Redis not available, skipping summary counter reset")
+        return
+
+    try:
+        redis_client = SUMMARY_STATUS_HANDLER._redis_client
+        redis_client.delete(REDIS_GLOBAL_SUMMARY_KEY)
+        logger.debug(
+            f"✅ Reset Redis summary counter '{REDIS_GLOBAL_SUMMARY_KEY}' "
+            f"at {SUMMARY_STATUS_HANDLER._redis_host}:{SUMMARY_STATUS_HANDLER._redis_port}"
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)